spec/features/authenticated_user/auth_non_admin_views_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.